### PR TITLE
Feature/pelagos 4055 update entity and tests

### PR DESCRIPTION
**Pull request by fightingtexasaggie**
We'll have the pre-migration copy of the database if for some unknown reason future GRIIDC would ever want these old attributes.

### CODE REVIEW

File: tests/unit/Entity/DatasetSubmissionTest.php [hunk 1]
```diff
@@ -348,40 +348,6 @@ public function testCanSetAndGetDatasetFileColdStorageAttributes()
         $this->assertNull($this->datasetSubmission->getDatasetFileColdStorageOriginalFilename());
     }
 
-    /**
```
Comment (praneethpr) [inline tests/unit/Entity/DatasetSubmissionTest.php:351]: I did a grep and these variables are used in ISOMetadataExtractorUtilTest

**Comment by mickel1138** [comment]
Reviewed files.

File: templates/MdApp/main.html.twig [hunk 2]
```diff
@@ -93,18 +93,7 @@
...
-                                                                onclick="javascript:showLogEntries('{{dta.udi}}')"
-                                                                id="log_butt_{{dta.udi}}">Log</button></td>
-                                        {% if dta.datasetsubmission.MetadataFileName|default %}
-                                            <td id="md_filename">{{ dta.udi }}</td>
-                                        {% else %}
-                                            <td id="md_filename">{{ dta.datasetsubmission.MetadataFileName|default }}</td>
-                                        {% endif %}
-                                    </tr>
-                                </table>
+                                <button style="font-size:66%" onclick="javascript:showLogEntries('{{dta.udi}}')" id="log_butt_{{dta.udi}}">Log</button></td>
```
Comment (mickel1138) [inline templates/MdApp/main.html.twig:96]: Simplified?

Comment (fightingtexasaggie) [reply templates/MdApp/main.html.twig:96]: yeah, this was old un-used cruft.

**Pull request merged**